api/mapping/forms.py

```diff
@@ -2,7 +2,7 @@
 from django.contrib.auth.forms import UserCreationForm
 from django.contrib.auth.models import User
 from django.core.exceptions import ValidationError
-from mapping.models import OmopTable, OmopField, DocumentType, DataPartner
+from mapping.models import OmopTable, OmopField, DocumentType, DataPartner,Document
```

Comment:
Please, format the code. On PyCharm, go to Menu Code > Optimize imports and Menu Code > Reformat Code.